api_proxy/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# How to build:
+# docker run --rm -e BUILD_GOOS="linux" -e BUILD_GOARCH="amd64" -v $(pwd):/src \
+#   -v /var/run/docker.sock:/var/run/docker.sock \
+#   centurylink/golang-builder-cross gcr.io/$DOCKER_NAMESPACE/api-proxy
```

Comment:
We should be commit hermetic dependencies to this repo.

Since go 1.5, you can always do `GOOS=linux GOARCH=amd64 go build -o api_proxy-linux-amd64 .` to build the binary, then `docker build`.

Am I missing something?